### ISSUE TITLE
Map folder not found when lazbuild from root repo folder

### ISSUE THREAD

**Issue by gcarreno** [wp-xyz/corona#43]
Hey Werner(@wp-xyz),

Could you please include the maps folder again?
It's probably because I'm at the tail spin of a very long day, but I could swear that the maps were included at some time...

Cheers,
Gus

**Comment by gcarreno** [comment]
Hey Werner(@wp-xyz),

**WAIT** Let me rephrase this!!!

It looks like the maps folder under the `*.lpi` is not relative or something.
If I `lazbuild source/corona.lpi` on the repo root folder it can't find the maps, but if I `cd source` and then `lazbuild corona.lpi` it will find the maps and no error is shown.

Alas the issue on #42 still persists

Cheers,
Gus

**Comment by wp-xyz** [comment]
The maps are still here, in source.

**Comment by gcarreno** [comment]
Hey Werner(@wp-xyz),

> The maps are still here, in source.

Yes they are and it's completely mea culpa for jumping the gun!!!

There is no issue if you are inside Lazarus or use `lazbuild` if you're on the `source` folder.
But if you use `lazbuild` from anywhere else, `maps` folder is not found.
Had a look at `corona.lpi` and the only thing I can think of is to use macros: `%ProjectDir%/maps/blabla`, but I'm not sure they work on the resource side of things.

Cheers,
Gus

**Comment by wp-xyz** [comment]
I never work with lazbuild, but if we had the corona.res, shouldn't it work always? ;-)

**Comment by gcarreno** [comment]
Hey Werner(@wp-xyz),

> I never work with lazbuild, but if we had the corona.res, shouldn't it work always? ;-)

Yeap, that's correct and it did show the map even if `lazbuild` complained about not finding the resources.

I don't think we can fix this since it's not possible to use IDE macros on the resources.

I'm closing it and I'll just always compile from the `source` folder.

Cheers,
Gus
